composer.json

```diff
@@ -24,6 +24,7 @@
   "require-dev": {
     "automattic/vipwpcs": "^2.1",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
+    "dg/bypass-finals": "1.1.3",
```

Comment:
Can you explain a little more why this package was added @ankitrox?

Reply:
@swissspidy is the license on this package compatible here?

Reply:
@spacedmonkey This package is added to mock the final classes in phpunit. In this case `WP_Customize_Manager`.